services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -48,12 +49,29 @@ class HankeServiceImpl(private val hankeRepository: HankeRepository) : HankeServ
      * Returns empty list if no items to return
      * TODO user information to limit what all Hanke items we get?
      */
-   override fun loadAllHanke(): List<Hanke> {
+    override fun loadAllHanke(): List<Hanke> {
 
         val entity = hankeRepository.findAll()
 
-        var hankeList:MutableList<Hanke>  = mutableListOf()
-        entity.forEach {  hankeEntity ->
+        val hankeList: MutableList<Hanke> = mutableListOf()
+        entity.forEach { hankeEntity ->
+            hankeList.add(createHankeDomainObjectFromEntity(hankeEntity))
+        }
+        return hankeList
+    }
+
+    /**
+     * Returns all the Hanke items from database with alkuPvm and/or loppuPvm between the periodStart and periodEnd
```

Comment:
... for which the hanke period overlaps with the given period.

Tuo nykyinen kuvaus ei vastaa sitä mitä tapahtuu jos alku on ennen periodStart ja loppu periodEnd jälkeen.